apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -334,9 +331,14 @@ public override Task<int> ExecuteNonQueryAsync(CancellationToken cancellationTok
             string commandText,
             CancellationToken cancellationToken)
         {
-            var databaseAdminClient = await DatabaseAdminClient.CreateAsync().ConfigureAwait(false);
+            var builder = spannerConnection.SpannerConnectionStringBuilder;
+            var channel = await SpannerClientFactory.CreateChannelAsync(
+                builder?.Host ?? DatabaseAdminClient.DefaultEndpoint.Host,
+                builder?.Port ?? DatabaseAdminClient.DefaultEndpoint.Port,
+                builder?.GetCredentials()).ConfigureAwait(false);
             try
             {
+                var databaseAdminClient = DatabaseAdminClient.Create(channel);
```

Comment:
I know nothing about this code, but why is a new channel/client being created for each DDL execution?

Reply:
It's simpler than trying to cache the clients or channels, basically. I think it's reasonable to expect that DDL won't be performance critical, so simplicity is good :)